packages/nocodb/src/lib/meta/api/metaDiffApis.ts

```diff
@@ -532,18 +539,225 @@ async function getMetaDiff(
 
 export async function metaDiff(req, res) {
   const project = await Project.getWithInfo(req.params.projectId);
-  const base = project.bases[0];
+  let changes = []
+  for (const base of project.bases) {
+    // @ts-ignore
+    const sqlClient = NcConnectionMgrv2.getSqlClient(base);
+    changes = changes.concat(await getMetaDiff(sqlClient, project, base));
```

Comment:
Handle exception with `try...catch` in the meta diff of each base to avoid blocking other bases(metal diff) from executing.